Sources/DistributedActors/Cluster/Reception/OperationLogDistributedReceptionist.swift

```diff
@@ -295,6 +295,22 @@ extension OpLogDistributedReceptionist: LifecycleWatch {
         }
     }
 
+    public nonisolated func checkIn<Guest>(
+        _ guest: Guest
+    ) async where Guest: DistributedActor, Guest.ActorSystem == ClusterSystem {
+        guard let keyID: String = guest.id.metadata.receptionID else {
+            fatalError("""
+            Attempted to \(#function) distributed actor without `@ActorID.Metadata(\\.receptionKey)` set on ActorID!
+            Please set the metadata during actor initialization.
+            """)
+        }
+        let key = DistributedReception.Key(Guest.self, id: keyID)
+
+        await self.whenLocal { myself in
```

Comment:
This reads very nicely 👍

Reply:
I recently realized we don't always have to do the `__` trickery by the way :)